src/libraries/Common/tests/System/DirectoryServices/LDAP.Configuration.xml

```diff
@@ -83,5 +83,14 @@ Note:
         <Password>%TESTPASSWORD%</Password>
         <AuthenticationTypes>ServerBind,None</AuthenticationTypes>
     </Connection>
+    <Connection Name="SLAPD OPENLDAP SERVER TLS">
```

Comment:
I'm thrilled that you're able to extend my refactored test system.

Can you add your osixia/openldap commands above in the comment, like the others? Or nobody will know how to hit this one.

Reply:
along with any instructions..

Reply:
> I'm thrilled that you're able to extend my refactored test system.

Oh good! I didn't know if I was supposed to do this or not.

I added some instructions, but they're kind of long.

Reply:
Looks perfect